main.py

```
@@ -1,44 +1,437 @@
+REGISTER_URL = "https://1weucj.life/?open=register&p=mlg1"
+AVIATOR_URL = "https://1weucj.life/?open=register&p=mlg1"
+
+bot = telebot.TeleBot(TOKEN)
+DB_FILE = "database.json"
+
+def load_db():
+    if os.path.exists(DB_FILE):
+        try:
+            with open(DB_FILE, 'r', encoding='utf-8') as f:
+                return json.load(f)
+        except: return {}
```

Ruff (0.15.12)

[error] Do not use bare `except`

(E722)